db/models/forms.py

```diff
@@ -0,0 +1,82 @@
+import uuid
+
+from db import db
+from db.models.applications import Applications
+from db.models.common import Status
+from sqlalchemy_json import NestedMutableJson
+from sqlalchemy_utils.types import UUIDType
+
+
+class Forms(db.Model):
+
+    id = db.Column(
+        "id",
+        UUIDType(binary=False),
+        default=uuid.uuid4,
+        primary_key=True,
+        nullable=False,
+    )
+
+    application_id = db.Column(
+        "application_id", db.ForeignKey(Applications.id), nullable=False
+    )
+
+    json = db.Column("json", NestedMutableJson)
+
+    status = db.Column(
+        "status", db.Enum(Status), default="NOT_STARTED", nullable=False
+    )
+
+    name = db.Column("name", db.String(), nullable=False)
+
```

Comment:
Could/should there be a unique constraint here on the two columns "application_id" and "name" so that an application can't have two forms with the same name? eg.
`__table_args__ = (UniqueConstraint('application_id', 'name', name='_form_name_uc'),)`

[Multiple column unique constraint on stackoverflow](https://stackoverflow.com/questions/10059345/sqlalchemy-unique-across-multiple-columns)